Driver build fails with kernel 5.15.xx

```
/usr/bin/git pull -v
From https://github.com/pguyot/tagtagtag-ears
 = [up to date]      release    -> origin/release
 = [up to date]      master     -> origin/master
Already up to date.
Building tagtagtag-ears driver for kernel 5.15.30+.
/usr/bin/make KERNELRELEASE=5.15.30+ clean && /usr/bin/make KERNELRELEASE=5.15.30+ && /usr/bin/sudo /usr/bin/make KERNELRELEASE=5.15.30+ install && /usr/bin/make KERNELRELEASE=5.15.30+ clean
make -C /lib/modules/5.15.30+/build M=/home/pi/tagtagtag-ears clean
make[1]: Entering directory '/usr/src/linux-headers-5.15.30+'
make[1]: Leaving directory '/usr/src/linux-headers-5.15.30+'
make -C /lib/modules/5.15.30+/build M=/home/pi/tagtagtag-ears modules
make[1]: Entering directory '/usr/src/linux-headers-5.15.30+'
  CC [M]  /home/pi/tagtagtag-ears/tagtagtag-ears.o
  MODPOST /home/pi/tagtagtag-ears/Module.symvers
  CC [M]  /home/pi/tagtagtag-ears/tagtagtag-ears.mod.o
  LD [M]  /home/pi/tagtagtag-ears/tagtagtag-ears.ko
make[1]: Leaving directory '/usr/src/linux-headers-5.15.30+'
make: *** No rule to make target 'tagtagtag-ears.dtbo', needed by 'install'.  Stop.
```

--- a/.github/workflows/arm-runner.yml
+++ b/.github/workflows/arm-runner.yml
@@ -6,7 +6,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        target: [zero_raspbian, zero_raspios, zero_dietpi, zero2_raspios, zero2_dietpi, zero2_64_raspios, zero2_64_dietpi]
+        target: [zero_raspbian, zero_raspios, zero_dietpi, zero2_raspios, zero2_dietpi, zero2_64_raspios, zero2_64_dietpi, pynab_latest]
         include:
         - target: zero_raspbian
           cpu: arm1176
@@ -36,6 +36,10 @@ jobs:
           cpu: cortex-a53
           cpu_info: cpuinfo/raspberrypi_zero2_w_arm64
           base_image: dietpi:rpi_armv8_bullseye
+        - target: pynab_latest
+          cpu: arm1176
+          cpu_info: cpuinfo/raspberrypi_zero_w
+          base_image: https://github.com/nabaztag2018/pynab/releases/download/v0.9.1/pynab-v0.9.1.img.xz
     steps:
     - uses: actions/checkout@v2
     - uses: pguyot/arm-runner-action@v2
@@ -45,8 +49,8 @@ jobs:
         cpu: ${{ matrix.cpu }}
         cpu_info: ${{ matrix.cpu_info }}
         commands: |
-            kernel_version=`apt list --installed | grep raspberrypi-kernel | awk '{ print $2 }' | sed -e 's|1:||g'`
-            arch=`apt list --installed | grep raspberrypi-kernel | awk '{ print $3 }'`
+            kernel_version=`apt list --installed | grep raspberrypi-kernel | awk 'NR == 1 { print $2 }' | sed -e 's|1:||g'`
+            arch=`apt list --installed | grep raspberrypi-kernel | awk 'NR == 1 { print $3 }'`
             wget -q http://archive.raspberrypi.org/debian/pool/main/r/raspberrypi-firmware/raspberrypi-kernel-headers_${kernel_version}_${arch}.deb
             dpkg --install raspberrypi-kernel-headers_${kernel_version}_${arch}.deb
             apt-get update -y --allow-releaseinfo-change

--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,13 @@ obj-m += tagtagtag-ears.o
 dtbo-y += tagtagtag-ears.dtbo
 
 targets += $(dtbo-y)
-always := $(dtbo-y)
+
+# Gracefully supporting the new always-y without cutting off older target with kernel 4.x
+ifeq ($(firstword $(subst ., ,$(KERNELRELEASE))),4)
+	always := $(dtbo-y)
+else
+	always-y := $(dtbo-y)
+endif
 
 all:
 	make -C /lib/modules/$(KERNELRELEASE)/build M=$(PWD) modules

